### PR TITLE
Exclude git from documentation artifact.

### PR DESCRIPTION
**Pull request by galanCA**
Exclude the git folder from the documentation artifact.

### CODE REVIEW

**Review by chipbarnaby — APPROVED** [review]
Pretty simple!

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -42,7 +42,9 @@ jobs:
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: doc/output
+          publish_dir: |
+                        doc/output
+                        !doc/output/.git
```
Comment (nealkruis) [inline .github/workflows/release.yml:47]: Are you sure you want to do this here?

Comment (galanCA) [reply .github/workflows/release.yml:47]: I was looking for a way to test that line but I didn't know-how. Is this not the same as build-and-test.yml?

**Review by nealkruis — APPROVED** [review]
Looks good. Thanks, @galanCA!

**Pull request merged**